tests/mapper/user_test.go

```diff
@@ -15,7 +15,7 @@ func TestUserMapper(t *testing.T) {
 	user := model.User{
 		Account:  "test",
 		Password: func() *string { s := "test"; return &s }(),
-		Roles:    []*model.Role{{Name: "admin"}},
+		// Roles:    []*model.Role{{Name: ""}}, // If added, foreign key constraint make it can't be delete.
```

Comment:
We should fix this, actually I remember there's something similiar with this problem in `problem` mapper.